rotkehlchen/chain/ethereum/modules/lido/decoder.py

```diff
@@ -16,6 +16,7 @@
 from rotkehlchen.types import ChecksumEvmAddress
 from rotkehlchen.utils.misc import bytes_to_address, from_wei
 
+from ..lido_csm.constants import CPT_LIDO_CSM
```

Comment:
```suggestion
from rotkehlchen.chain.ethereum.modules.lido_csm.constants import CPT_LIDO_CSM
```
Better to use the full path instead of `..`